### ISSUE TITLE
[11.0] mail.activity calendar view

### ISSUE THREAD

**Issue by aitorbouzas** [odoo/odoo#33541]
Impacted versions:
11.0

Steps to reproduce:
Get inside calendar view for mail.activity. It shows create_uid instead of user_id.

Current behavior:
Shows the user that created the activity.
https://github.com/odoo/odoo/blob/7237b8f2a223204792b1a9ff3a12719675fbc558/addons/mail/views/mail_activity_views.xml#L158

Expected behavior:
Show the user that's assigned to the activity
```
<field name="user_id"/>
```

Video/Screenshot link (optional):

**Comment by Yenthe666** [comment]
Hmmm. There is something to be said about both choices honestly.
@tde-banana-odoo could you be so kind to give your final call on this?

**Comment by aitorbouzas** [comment]
Do you think there's a reason to keep `create_uid`?
I'm trying to watch this from a sales team manager perspective, trying to check what activities his team is planning and if they are up to date...

**Comment by Yenthe666** [comment]
Yeah because then you know who created the activity and who is assigned. That way you can figure out who has assigned which person for an activity.
